linearizable/scripts/recreatedb

```diff
@@ -5,6 +5,18 @@ if [[ -n "$1" ]]; then
     shift
 fi
 
+lrlpath=""
+if [[ -n "$1" ]]; then
+    lrlpath=$1
+    shift
+fi
+
+add_tables=1
+if [[ "$1" == "noadd" ]]; then
```

Comment:
It was a convenience script I used while testing for linearizability.  I'm now moving these tests to our normal testsuite so this will be removed very soon.